zerver/lib/email_mirror.py

```diff
@@ -203,7 +203,8 @@ def send_zulip(sender, stream, topic, content):
         "stream",
         stream.name,
         topic[:60],
-        content[:2000])
+        content[:2000],
+        True)
```

Comment:
Spell this out as `email_mirror=True`.  In general, we don't like to pass kwargs like this as positional arguments, since it makes it harder to find things with grep etc.